style.css

```diff
@@ -1,468 +1,1187 @@
+  display: flex;
+  padding-left: 10px;
+  padding-top: 6px;
+  font-weight: 600;
+  font-size: 35px;
+  color: #000000;
 }
-.speed__number {
-    padding-left: 8px;
-    font-weight: 600;
-    font-size: 35px;
-    color: #000000;
+
+.information__number-subtitle {
+  padding-top: 13px;
+  padding-left: 8px;
+  font-weight: 600;
+  font-size: 20px;
+  color: #000000;
 }
-.speed__text {
-    padding-top: 14px;
-    padding-left: 7px;
-    font-weight: 600;
-    font-size: 20px;
-    color: #000000;
-}
-.speed__desc {
-    padding-top: 41px;
-    margin-left: -98px;
-    font-style: normal;
-    font-weight: normal;
-    font-size: 12px;
-    line-height: 15px;
-    color: #000000;
-}
-.test {
-    display: flex;
-    padding-left: 796px;
-    margin-top: -45px;
+
+.information__number-text {
+  font-size: 12px;
+  padding-left: 10px;
+  margin-top: -3px;
 }
-.button {
-    width: 160px;
-    height: 40px;
-    margin-left: 959px;
-    margin-top: -38px;
-    border: 2px solid #E51937;
-    box-sizing: border-box;
+
+.speed__foto {
+  padding-top: 9px;
 }
+
+.test__foto {
+  padding-left: 16px;
+  padding-top: 7px;
+}
+
+.information__block__button {
+  width: 160px;
+  height: 40px;
+  margin-right: 5px;
+  margin-top: 17px;
+  border: 2px solid #e51937;
+  box-sizing: border-box;
+}
+
 .button__text {
-    font-weight: bold;
-    font-size: 13px;
-    text-transform: uppercase;
-    color: #E51937;
-    background: transparent;
-    padding-top: 9px;
-    padding-left: 41.5px;
-    letter-spacing: .8px;
+  font-weight: bold;
+  font-size: 13px;
+  text-transform: uppercase;
+  color: #e51937;
+  background: transparent;
+  margin-top: -18px;
+  padding-left: 41.5px;
+  letter-spacing: 0.8px;
 }
+
 .section-first__arrow {
-    width: 28px;
-    height: 14px;
-    padding-left: 703px;
-    padding-top: 40px;
+  width: 28px;
+  height: 14px;
+  padding-top: 26px;
+  margin: 0 auto;
 }
 
 /*Style section-second*/
+
 .section-second {
-    /*position: absolute;*/
-    width: 1548px;
-    height: 844px;
-    left: -275px;
-    top: 1056px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  background-image: url("image/2013-tesla-model-s-cockpit.png");
+  background-position: -5px 107px;
+  background-repeat: no-repeat;
+  background-size: 89%;
 }
+
 .section-second__title {
-    margin-top: 360px;
+  width: 31%;
+  text-align: center;
+  padding-top: 118px;
+  font-weight: 600;
+  font-size: 70px;
+  line-height: 85px;
+  color: #e51937;
+}
+
+.aside {
+  width: 440px;
+  margin-top: -146px;
+  margin-left: auto;
+  padding-top: 52px;
+  padding-left: 47px;
+  padding-bottom: 23px;
+  background: #e51937;
+}
+
+.aside__title {
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  padding-left: 5px;
+  color: #fff;
+}
+
+.aside__content {
+  width: 316px;
+  height: 206px;
+  padding-top: 11px;
+  font-weight: 500;
+  font-size: 15px;
+  margin-bottom: 44px;
+  padding-left: 5px;
+  color: #ffffff;
```

Comment:
Codacy found an issue: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Project_AnastassiaVarabei_Evgen/pullRequest?prid=6131199)